src/components/Address/README.md

```diff
@@ -0,0 +1,34 @@
+# Ethereum Address component
+
+![Default component vie](./doc/basic.png)
+
+![Default component vie](./doc/reverse.png)
+
+Please refer to `src/routes/Test.js` for examples.
+
+Demo: http://localhost:3000/Test
+E2E test: `cypress/integration/address.spec.js`
+
+### Basic usage
+
+```
+<AddressInput provider={web3.currentProvider} />
+```
+
+#### Parameters:
+
+- provider - Web3 provider **(required)**
+- placeholder - set custom placeholder. Default: 'Enter Ethereum name or address'
+- showBlockies - show digital image based on address (similar to github). Default: true
+- showSearchIcon - search icon on the left. Only present when nothing is typed. Default: true
+- onError - callback. Invokes every time error is happened or invalid name is typed. When typed name is corrected, invokes with null after error.
```

Comment:
How do I test this?

Reply:
Testing unnecessary for this for now - this is for when we release it as a separate package.